src/index.js

```diff
@@ -139,64 +154,51 @@ function expectFiles (paths, noFilesFn) {
     }
 }
 
-function readFiles (paths) {
-    paths.forEach(function (p) {
-        var stat = fs.statSync(p);
+function readFiles (paths, cb) {
```

Comment:
I think this function is improperly named now. The actual reading is being done further up.